themes/theme-ios11/components/Viewport/index.jsx

```diff
@@ -5,25 +5,58 @@ import TabBar from 'Components/TabBar';
 import styles from './style';
 
 /**
- * The Viewport component.
- * @param {Object} props The component props.
- * @returns {JSX}
+ * The view component.
```

Comment:
The issue still occurs for me. I think, it doesn't occur for theme pages because we use `react-loadable` to load the page components there. With this module the components are rendered in a different order for theme pages. So at the time when the AppBar renders for those pages, the portal container is already added to the DOM.

For custom pages, the page component renders within the normal call stack, and the container is not added yet.

Reply:
For me the problem didn't occur anymore. I tested it with a customer extension that provides custom routes.